example/vanilla-k8s-RWO-filesystem-volumes/example-sc-WaitForFirstConsumer-restricted.yaml

```diff
@@ -9,10 +9,10 @@ parameters:
   storagepolicyname: "vSAN Default Storage Policy"  # Optional Parameter
 allowedTopologies:
   - matchLabelExpressions:
-      - key: topology.kubernetes.io/zone
```

Comment:
deployment instruction uses `k8s-region` and `k8s-zone` as topology category.
Do you want to chagne that as well?

Reply:
Done